hazelcast/src/hazelcast/client/proxy/ClientPNCounterProxy.cpp

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <boost/foreach.hpp>
+
+#include "hazelcast/client/protocol/codec/PNCounterAddCodec.h"
+#include "hazelcast/client/protocol/codec/PNCounterGetConfiguredReplicaCountCodec.h"
+#include "hazelcast/client/cluster/memberselector/MemberSelectors.h"
+#include "hazelcast/client/proxy/ClientPNCounterProxy.h"
+#include "hazelcast/client/protocol/codec/PNCounterGetCodec.h"
+#include "hazelcast/client/spi/ClientClusterService.h"
+#include "hazelcast/client/cluster/impl/VectorClock.h"
+#include "hazelcast/util/Util.h"
+
+namespace hazelcast {
+    namespace client {
+        namespace proxy {
+            const std::string ClientPNCounterProxy::SERVICE_NAME = "hz:impl:PNCounterService";
+            const boost::shared_ptr<std::set<Address> > ClientPNCounterProxy::EMPTY_ADDRESS_LIST(
+                    new std::set<Address>());
+
+            ClientPNCounterProxy::ClientPNCounterProxy(const std::string &serviceName, const std::string &objectName,
+                                                       spi::ClientContext *context) : ProxyImpl(serviceName, objectName,
+                                                                                                context),
```

Comment:
Can't you make this more human readable ?

Reply:
Ok, changed.